### ISSUE TITLE
Add support to run Unity Test Tools Runner on Travis CI

### ISSUE THREAD

**Issue by mlnv** [GandaG/unityCItest#19]
It would be nice to check that all tests passed after pull changes to git repo.

**Comment by dh-nunes** [comment]
I don't understand, do you mean testing that the code runs properly?

**Comment by mlnv** [comment]
Yes, run tests that handle Unity-Test-Tools asset and show info in Travis CI. Here is documentation https://bitbucket.org/Unity-Technologies/unitytesttools/wiki/Home 
That would be awesome! Also thanks for this awesome repo

**Comment by dh-nunes** [comment]
Oh, that seems pretty easy actually. I won't have time for at least 3 weeks but I'll be sure to implement it as soon as I can!
